src/pages/Video.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { useVideo } from "../context/video-context";
+import "../style/video.css";
+import { useParams } from "react-router-dom";
+import { useLike } from "../context/like-context";
+import { usePlaylist } from "../context/playlist-context";
+import { useWatchLater } from "../context/watchlater-context";
+import { PlaylistModal } from "../component";
+import Skeleton from "react-loading-skeleton";
+import "react-loading-skeleton/dist/skeleton.css";
+
+export function Video() {
+  const { videoList } = useVideo();
+  const { likeDispatch } = useLike();
+  const { watchDispatch } = useWatchLater();
+  const { showModal, setShowModal } = usePlaylist();
+
+  let { _id } = useParams();
+  let videoDetails = videoList.find((ele) => ele._id === _id);
```

Comment:
you can write a function presentIn array which will do the find. This is just a suggestion because you are using this at multiple places

Reply:
okay